### PR TITLE
HUE-9644 [Django Upgrade] Port remaining apps to proper URL namespacing

### PR DESCRIPTION
**Pull request by agl29**

### CODE REVIEW

File: docs/release-notes/release-notes-2.0.0-beta.txt [hunk 1]
```diff
@@ -357,7 +357,7 @@ List of All Commits
 * HUE-633. [jobsub] Port examples to new design models
 * [test] Work around a webhdfs redirect bug in test
 * [doc] document dependency on libldap2-dev
-* HUE-648. [fb] filebrowser.views.view() makes too many filesystem calls
+* HUE-648. [fb] filebrowser:filebrowser.views.view() makes too many filesystem calls
```
Comment (romainr) [inline docs/release-notes/release-notes-2.0.0-beta.txt:360]: nit: not needed

same below

**Review by romainr — APPROVED** [review]
Nice one!

And does a fresh build still works?

e.g.

fresh clone of you branch or
`make clean; make apps`

**Comment by agl29** [comment]
> Nice one!
> 
> And does a fresh build still works?
> 
> e.g.
> 
> fresh clone of you branch or
> `make clean; make apps`

yes, after doing a fresh build it is working.

**Comment by agl29** [comment]
https://app.circleci.com/pipelines/github/cloudera/hue/3976/workflows/19ab861a-18ed-4dda-b050-e4fe967bdfe7/jobs/5383 these are the 2 pytests that are failing. (I have checked that test locally, they running fine i.e. no failure)

File: docs/docs-site/content/releases/release-notes-2.0.0-beta.md [hunk 2]
```diff
@@ -365,7 +365,7 @@ List of All Commits
 * HUE-633. [jobsub] Port examples to new design models
 * [test] Work around a webhdfs redirect bug in test
 * [doc] document dependency on libldap2-dev
-* HUE-648. [fb] filebrowser:filebrowser.views.view() makes too many filesystem calls
+* HUE-648. [fb] filebrowser.views.view() makes too many filesystem calls
```
Comment (romainr) [inline docs/docs-site/content/releases/release-notes-2.0.0-beta.md]: 4 docs changes to avoid as not true at the time

**Review by romainr — APPROVED** [review]
Better commit message than 'Pylint issues?

**Comment by agl29** [comment]
Screenshot 2020-12-16 at 9 23 47 PM
Not understanding why the useradmin tests like useradmin.tests.TestUserAdmin.test_group_permissions failing. Although I can see the permission page. Any hint?

**Comment by romainr** [comment]
File "/usr/share/hue/apps/useradmin/src/useradmin/views.py", line 408, in edit_user
    return redirect(reverse(list_users))

`current_app` needed? https://docs.djangoproject.com/en/3.1/ref/urlresolvers/#reverse

**Comment by agl29** [comment]
But as written that 
# passing a callable object
# (This is discouraged because you can't reverse namespaced views this way.)
Screenshot 2020-12-16 at 11 25 49 PM

**Comment by romainr** [comment]
But actually (just saw the comment above), indeed, should just switch to the string '..useradmin.views.list_user' if no namespace support!

**Comment by agl29** [comment]
This is the same thing I was thinking but after the change does it support all things like in this https://github.com/cloudera/hue/blob/master/apps/useradmin/src/useradmin/views.py#L93 method there is a if-else condition so how we ensure that thing.

**Comment by romainr** [comment]
Let's just use the full namespace string name and not the Python view

**Comment by agl29** [comment]
> Let's just use the full namespace string name and not the Python view

Done.

File: docs/docs-site/content/releases/release-notes-2.0.0-beta.md (hunk 2 above)
Comment (romainr) [reply docs/docs-site/content/releases/release-notes-2.0.0-beta.md]: ping

**Review by romainr — APPROVED** [review]

File: docs/docs-site/content/releases/release-notes-2.0.0-beta.md (hunk 2 above)
Comment (agl29) [reply docs/docs-site/content/releases/release-notes-2.0.0-beta.md]: what thing I have to do here I am not understanding? @romainr

**Pull request merged**